### ISSUE TITLE
image usage

### ISSUE THREAD

**Issue by A-transformer** [Project-HAMi/HAMi-core#21]
"After building the Docker image, we copy the build package into the target image. Does this mean that only our contain build package image will support the following features?
on 
export CUDA_DEVICE_MEMORY_LIMIT=1g
export CUDA_DEVICE_SM_LIMIT=50

**Comment by archlitchi** [comment]
actually, it has nothing to do with 'image', the output of build is 'libvgpu.so', you can use that wherever you want with LD_PRELOAD

**Comment by A-transformer** [comment]
Yes, it's working on both container and VM environments. Is it possible to set CUDA_DEVICE_MEMORY_LIMIT and CUDA_DEVICE_SM_LIMIT dynamically? Once the process starts, the environment variables become fixed. Can these values be changed dynamically or reloaded from a ConfigMap or configuration server?

**Comment by archlitchi** [comment]
> Yes, it's working on both container and VM environments. Is it possible to set CUDA_DEVICE_MEMORY_LIMIT and CUDA_DEVICE_SM_LIMIT dynamically? Once the process starts, the environment variables become fixed. Can these values be changed dynamically or reloaded from a ConfigMap or configuration server?

We haven't implemented that feature, but technically, you can modify values in vGPUmonitor, once the '.cache' file in mmaped, it will update device memory limit and utilization limit to the corresponding container.

**Comment by mesutoezdil** [comment]
Hi,

We are going through open issues to keep things manageable. This issue was opened before 2026 and has not been resolved or updated recently.

Please reply by 2026-07-13 to confirm this is still relevant and based on the current codebase. If we do not hear back, we will close this issue. You are welcome to reopen it or submit a new one when ready.

Thank you for your contribution to HAMi.
